Python/Auto_Caption_Generator/requirements.txt

```diff
@@ -0,0 +1,114 @@
+absl-py==0.11.0
+appdirs==1.4.4
+argon2-cffi==20.1.0
+astunparse==1.6.3
+async-generator==1.10
+attrs==20.3.0
+audioread==2.1.9
+backcall==0.2.0
+bleach==3.2.1
+cachetools==4.2.0
+certifi==2020.12.5
+cffi==1.14.4
+chardet==4.0.0
+colorama==0.4.4
+cycler==0.10.0
+decorator==4.4.2
+defusedxml==0.6.0
+entrypoints==0.3
+flatbuffers==1.12
+gast==0.3.3
+google-auth==1.24.0
+google-auth-oauthlib==0.4.2
+google-pasta==0.2.0
+grpcio==1.32.0
+h5py==2.10.0
+idna==2.10
+ipykernel==5.4.3
+ipython==7.19.0
+ipython-genutils==0.2.0
+ipywidgets==7.6.3
+jedi==0.18.0
+Jinja2==2.11.2
+joblib==1.0.0
+jsonschema==3.2.0
+jupyter==1.0.0
+jupyter-client==6.1.11
+jupyter-console==6.2.0
+jupyter-core==4.7.0
+jupyter-http-over-ws==0.0.8
+jupyterlab-pygments==0.1.2
+jupyterlab-widgets==1.0.0
+Keras==2.4.3
+Keras-Preprocessing==1.1.2
+kiwisolver==1.3.1
+librosa==0.8.0
+llvmlite==0.35.0
+Markdown==3.3.3
+MarkupSafe==1.1.1
+matplotlib==3.3.3
+mistune==0.8.4
+nbclient==0.5.1
+nbconvert==6.0.7
+nbformat==5.1.2
+nest-asyncio==1.4.3
+notebook==6.2.0
+numba==0.52.0
+numpy==1.19.5
+oauthlib==3.1.0
+opt-einsum==3.3.0
+packaging==20.8
+pandas==1.2.0
+pandocfilters==1.4.3
+parso==0.8.1
+pickleshare==0.7.5
+Pillow==8.0.1
+pooch==1.3.0
+prometheus-client==0.9.0
```

Comment:
Why are we using `prometheus-client` here :sweat_smile: 

Can you use this [PyPi Library](https://pypi.org/project/pip-check-reqs/) for the same?

Reply:
That is one interesting Lib, I removed more than half of useless dependencies. Thanks!!!